src/systemctl/systemctl-logind.c

```diff
@@ -226,6 +237,11 @@ int logind_check_inhibitors(enum action a) {
         if (c <= 0)
                 return 0;
 
+        if (enforce_inhibitors)
+                return log_error_errno(
+                                SYNTHETIC_ERRNO(EPERM),
+                                "Please retry operation after closing inhibitors and logging out other users.");
```

Comment:
Please drop this then. The original error text is more informative on `--check-inhibitors` option.

Reply:
The client-side check does not apply when enforced mode is enabled, that's why I restructured it as such.